src/Host/Client/Impl/BrokerServices/RemoteUriWebService.cs

```diff
@@ -212,5 +214,13 @@ public class RemoteUriWebService : IRemoteUriWebService {
                 httpListenerResponse.Headers.Add(pair.Key, pair.Value);
             }
         }
+
+        private bool ValidateCertificate(object sender, X509Certificate certificate, X509Chain chain, SslPolicyErrors sslPolicyErrors) {
+            if ((sslPolicyErrors & SslPolicyErrors.RemoteCertificateNotAvailable) != 0) {
+                return false;
+            }
+            // Accept other cases. Validation for untrusted certificates is done by the broker.
```

Comment:
Wrong comment. I meant certificate validation is done at the time we connect to the broker. This service does not need to go to the same extent.

---
In reply to: [82295549](https://github.com/Microsoft/RTVS/pull/2447#discussion_r82295549) [](ancestors = 82295549)